packages/backend/src/config/Config.ts

```diff
@@ -56,19 +56,17 @@ export interface TvlConfig {
   readonly detailedTvlEnabled: boolean
   readonly errorOnUnsyncedDetailedTvl: boolean
   readonly coingeckoApiKey: string | undefined
-  readonly ethereum: EthereumTvlConfig | false
-  readonly arbitrum: ArbitrumTvlConfig | false
+  readonly ethereum: ChainTvlConfig | false
+  readonly arbitrum: ChainTvlConfig | false
+  readonly optimism: ChainTvlConfig | false
 }
 
-export interface EthereumTvlConfig {
+export interface ChainTvlConfig {
   readonly providerUrl: string
+  readonly providerCallsPerMinute: number
+  readonly networkName: string
```

Comment:
this one is not needed probably + in the case of base ethers do not have it as a config option